tracer/tracer.go

```diff
@@ -542,9 +550,15 @@ func loadPerfUnwinders(coll *cebpf.CollectionSpec, ebpfProgs map[string]*cebpf.P
 
 	progs := make([]progLoaderHelper, len(tailCallProgs)+2)
 	copy(progs, tailCallProgs)
+
+	sched_process_free := "tracepoint__sched_process_free"
+	if _, ok := coll.Programs["tracepoint__sched_process_free_old"]; ok {
+		sched_process_free = "tracepoint__sched_process_free_old"
+	}
```

Comment:
This also seems somewhat clumsy. Could this function get as an argument the tracepoint name instead?